practice/01-js-intro-and-jquery/Exercise1-javascript and jquery/js/scripts.js

```diff
@@ -11,16 +11,34 @@ $('document').ready(function(){
 	$('.alias').css('border-color','red');
 
 	$("#acept").click(function(){
-        $.ajax({url: "http://bootcamp.aws.af.cm/welcome/yourname", success: function(result){
-            //$("#div1").html(result);
-            console.log(result);
-            var cambio=JSON.stringify(result);
-            alert(cambio);
-        }});
+        var name="Marcos";
+        $.ajax({url: "http://bootcamp.aws.af.cm/welcome/"+name, success: function(result){
```

Comment:
I cant find fault, but I see you do not use the ajax function correctly. There are many ways to make the call ajax.
Example:
```
$.ajax( "example.php" )
  .done(function() {
    alert( "success" );
  })
  .fail(function() {
    alert( "error" );
  })
  .always(function() {
    alert( "complete" );
  });
```
wich equals:

```
$.ajax( "example.php" ).done(function() { alert( "success" ); }).fail(function() { alert( "error" ); }).always(function() {    alert( "complete" ); });
```
Also saw the result (which is json) receive it , transform it , and cut it . That's wrong , because you should take it as json . With json is easier.
Example:

```
$.ajax( "example.php",'json' ).done(function(result) {
     console.log(result); // you see one object. Result is json.
     console.log(result.response); //you see text
      $('#div-id').html(result.response);
 });
```